python/lsst/ts/xml/data/sal_interfaces/GIS/GIS_Events.xml

```diff
@@ -31,4 +31,1674 @@
+      <EFDB_Name>sdoCPUfree8</EFDB_Name>
+      <Description>Reserve 8.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdoCPUfree9</EFDB_Name>
+      <Description>Reserve 9</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdoCPUfree10</EFDB_Name>
+      <Description>Reserve 10.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdoCPUfree11</EFDB_Name>
+      <Description>Reserve 11.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdoCPUfree12</EFDB_Name>
+      <Description>Reserve 12.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>GIS</Subsystem>
+    <EFDB_Topic>GIS_logevent_gisCpuReserve</EFDB_Topic>
+    <item>
+      <EFDB_Name>sdrCPUfree</EFDB_Name>
+      <Description>Reserve.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>16</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>GIS</Subsystem>
+    <EFDB_Topic>GIS_logevent_afeDecentralizedIOInputs</EFDB_Topic>
+    <item>
+      <EFDB_Name>sdiAFEetacA</EFDB_Name>
+      <Description>Emergency channel 1 from access control.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdiAFEetacB</EFDB_Name>
+      <Description>Emergency channel 2 from access control.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdiAFEetsfA</EFDB_Name>
+      <Description>Emergency channel 1 from summit fire control.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdiAFEetsfB</EFDB_Name>
+      <Description>Emergency channel 2 from summit fire control.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdiAFEeteaA</EFDB_Name>
+      <Description>Emergency channel 1 from earthquake control.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdiAFEeteaB</EFDB_Name>
+      <Description>Emergency channel 2 from earthquake control.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdiAFEupieraA</EFDB_Name>
+      <Description>Unauthorized Pier Access channel 1.</Description>
```

Comment:
Please start all words except the first with a lower case character, here and everywhere else.